framework/src/transfers/MultiAppCopyTransfer.C

```diff
@@ -51,204 +52,99 @@ MultiAppCopyTransfer::initialSetup()
 }
 
 void
-MultiAppCopyTransfer::execute()
+MultiAppCopyTransfer::transferDofObject()
 {
-  _console << "Beginning CopyTransfer " << name() << std::endl;
-
-  switch (_direction)
-  {
-    case TO_MULTIAPP:
+  if (_to_object->n_dofs(_to_sys_num, _to_var_num) > 0) // If this variable has dofs at this node
```

Comment:
ew.

Pass in the `to_object` and the `from_object`... don't set them through class member variables like this... just kind of "icky"